### DISCUSSION TITLE
Connect to network by default

### DISCUSSION THREAD

**Discussion by tristantarrant** [containers/podman#15024] [Q&A] [answered]
Running a rootless podman (version 4.0.2), how can I make it connect to the default network without having to specify the `--network` argument every time ?

Running
```
podman run -it --name ubi --rm ubi9-micro /bin/bash
```
and then inspecting its IP address:
```
$ podman inspect -f "{{ .NetworkSettings.IPAddress }}" ubi

```

Running
```
podman run -it --network podman --name ubi --rm ubi9-micro /bin/bash
```
and then inspecting its IP address:
```
$ podman inspect -f "{{ .NetworkSettings.IPAddress }}" ubi
10.88.0.4
```

I have tried "forcing" the default network:
```
cat $HOME/.config/containers/containers.conf 
[network]
default_network="podman"
```
to no avail.

Is it possible ?

**Comment by mheon** [comment]
@Luap99 Did we fix this in 4.1? I recall changes being made to make bridge the default for rootless?

**Reply by Luap99** [reply]
Yes that should be fixed already, not sure what version. Definitely on main.

**Reply by Luap99** [reply]
Oh the default was only changed for docker API endpoints not podman

**Comment by Luap99** [comment, marked as answer]
You have to change the default netns to bridge in containers.conf

**Reply by tristantarrant** [reply]
Great that works using the CLI. 
For containers created via the docker API I guess I must upgrade to 4.1 ?

**Reply by Luap99** [reply]
yes, I think so
